scripts/delayed.js

```diff
@@ -7,11 +7,11 @@ import {
  * Swoosh on page
  */
 function pageSwoosh() {
-  const showSwoosh = getMetadata('show-swoosh');
-  if (showSwoosh === 'true') {
-    document.body.classList.add('page-swoosh');
+  const pSwoosh = getMetadata('page-swoosh');
+  if (pSwoosh && pSwoosh !== 'page-swoosh-no') {
+    document.body.classList.add(pSwoosh);
   } else {
-    document.body.classList.remove('page-swoosh');
+    document.body.classList.remove(pSwoosh);
```

Comment:
I feel delayed phase might be too late. Maybe we can load this in the lazy phase. @mkbansal1 lmk what you thinkl.